cozy/synthesis/acceleration.py

```diff
@@ -199,6 +202,9 @@ def inline_mapget(e : EMapGet, context : Context) -> Exp:
     print("warning: unable to inline {}".format(e))
     return e
 
+def statevar(e, ty):
+    return EStateVar(e.with_type(ty)).with_type(ty)
```

Comment:
I'm a little surprised there isn't an identical helper already defined somewhere. If you wanted, you could move this to `target_syntax.py` alongside the other helpers there. It is also fine to leave it here.